web/src/tippyjs.ts

```diff
@@ -502,10 +503,24 @@ export function initialize(): void {
     });
 
     delegate("body", {
-        target: "#user_info_popover .status-emoji",
+        target: ".user-card-status-text .status-emoji",
         appendTo: () => document.body,
     });
 
+    delegate("body", {
+        target: ".status-emoji-name",
+        placement: "bottom",
+        delay: INSTANT_HOVER_DELAY,
+        appendTo: () => document.body,
+        onShow() {
+            popovers.hide_all();
+        },
+
+        onHidden(instance) {
+            instance.destroy();
+        },
+    });
```

Comment:
So this solution covers all the locations other than the sidebars; can you add some comments both here and in `click_handlers.js` mentioning that the other code paths exist, and explaining why we need the sidebar-specific logic?

I think what you have is probably correct but we should document it for posterity.

Reply:
Sure , added the comments explaining why we need a different logic for sidebars.
PTAL.